index.html

```diff
@@ -237,6 +239,12 @@ <h2>Scope</h2>
 that are conformant with the data model and syntax(es).
           </li>
           <li>
+Recommend a conformance rubric of decentralized characteristics for DID Method
+specifications. This allows the DID Method specs to self-certify, or
+independent third parties to evaluate, the DID Method spec's level of
```

Comment:
"spec's" -> "specification's

Reply:
addressed in [b3d5a0e](https://github.com/w3c-ccg/did-wg-charter/pull/28/commits/b3d5a0ecb1a19c3405195345fde41316901543f2)